OpenRA.Mods.Common/Traits/Conditions/GrantConditionOnDeploy.cs

```diff
@@ -54,6 +54,9 @@ public class GrantConditionOnDeployInfo : PausableConditionalTraitInfo
 		[Desc("Skip make/deploy animation?")]
 		public readonly bool SkipMakeAnimation = false;
 
+		[Desc("Can this actor be ordered to move when deployed? [always, never, force-only]")]
+		public readonly string UndeployOnMove = "always";
```

Comment:
Always prefer enums to magic strings - the `FieldLoader` can automatically turn `Always` in the yaml into `UndeployType.Always` (or whatever we want to call the enum and values).

Reply:
Turned `UndeployOnMove` into an enum.